Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -107,44 +119,43 @@ for module in "${HYPERV_MODULES[@]}"; do
 
     # Check to see if the module is loaded
     if [[ $load_status =~ $module ]]; then
-        if [ "$rpmAvailable" = true ] ; then
-            if [ ! -z "$isLISInstalled" ]; then
-                version=$(modinfo "$module" | grep version: | head -1 | awk '{print $2}')
-                LogMsg "$module module: ${version}"
-                if [ "$module" == "mlx4_en" ] ;then
-                    if [ "$MLNX_VERSION" != "$version" ] ;then
-                        LogErr "ERROR: Status: $module $version doesnot match with build version $MLNX_VERSION"
-                        exit_code="1"
-                    fi
-                    continue
-                fi
-                if [ "$expected_lis_version" != "$version" ] ;then
-                    LogErr "ERROR: Status: $module $version doesnot match with build version $expected_lis_version"
-                    exit_code="1"
+        LogMsg "LIS nmodule $module loaded successfully"
+        if [ "$rpmAvailable" = true ] && [ ! -z "$isLISInstalled" ]; then
+            version=$(modinfo "$module" | grep version: | head -1 | awk '{print $2}')
+            LogMsg "$module module version: ${version}"
+            if [ "$module" == "mlx4_en" ] ;then
+                if [ "$MLNX_VERSION" != "$version" ] ;then
```

Comment:
We should add this condition in the previous if condition. is there any particular reason of keeping it separate?

Reply:
It looks it verifies LIS built-in version for all other modules except mlx4. I sort out the condition again here.